process/block/baseProcess.go

```diff
@@ -483,3 +479,19 @@ func checkProcessorNilParameters(
 
 	return nil
 }
+
+func (bp *baseProcessor) displayLastNotarized(lastNotarizedHdrForShard data.HeaderHandler, shardId uint32) {
```

Comment:
can't this function be in the display related file?

Reply:
Moved it